File: 프로그래머스/0/181870. ad 제거하기/ad 제거하기.java
class Solution {
    public String[] solution(String[] strArr) {
        int count = 0;
        String n = "ad";
        boolean contain = true;
        for(int i = 0; i < strArr.length; i++){
            contain = strArr[i].contains(n);
            if(contain){
                count++;
            }
        }
        String[] answer = new String[strArr.length-count];
        count = 0;
        for(int i = 0; i < strArr.length; i++){
            contain = strArr[i].contains(n);
            if(contain){
            }
            else{
                answer[count++] = strArr[i];
            }
        }
        return answer;
    }
}

File: 프로그래머스/0/181870. ad 제거하기/ad 제거하기.cs
using System;

public class Solution {
    public string[] solution(string[] strArr) {
        int count = 0;
        string n = "ad";
        
        for (int i = 0; i < strArr.Length; i++) {
            if (strArr[i].Contains(n)) {
                count++;
            }
        }
        
        string[] answer = new string[strArr.Length - count];
        count = 0;
        
        for (int i = 0; i < strArr.Length; i++) {
            if (!strArr[i].Contains(n)) {
                answer[count++] = strArr[i];
            }
        }
        
        return answer;
    }
}
